opencraft/settings.py

```diff
@@ -192,6 +192,8 @@
 
 STATIC_ROOT = root('build/static')
 STATIC_URL = '/static/'
+MEDIA_ROOT = root('media')
+MEDIA_URL = '/media/'
```

Comment:
Isn't this change missing the `urlpatterns` entry too?

Reply:
I added `ENABLE_DEBUG_TOOLBAR=true` to my `.env`, so it was automatically added [here](https://github.com/open-craft/opencraft/blob/master/opencraft/urls.py#L52). Not sure if we have a common `.env` file for development devstack. If so, it might be good to add it there.

Reply:
Btw, can you add a conditional here? Only set these values if `DEBUG=True`, otherwise this might mess up the S3 stage and production configuration.

Reply:
Sure, done.